app/views/MessagesView/index.tsx

```diff
@@ -227,7 +257,7 @@ class MessagesView extends React.Component {
 		}
 	};
 
-	getCustomEmoji = name => {
+	getCustomEmoji = (name: any) => {
```

Comment:
`name: { [name: string]: string }` ?

Reply:
to resolve that, I needed change the `customEmojis` to receive an index signature... 
the code was like this: `customEmojis: { [key: string]: string };` and the `getCustomEmoji = (name: string) => {`